pages/integrations/sso/custom_saml.md.erb

```diff
@@ -4,22 +4,35 @@ Any identify provider that supports SAML 2.0 can be used to authorize access to
 
 <%= toc %>
 
-## Set up your provider
+## Step 1. Set up your provider
 
-There are two ways to set up your custom provider: using the Buildkite meta-data XML to your identity provider, or manually adding your Buildkite data into your identity provider. After setting up your provider, you'll need to set up your SSO provider in your Buildkite organization's Single Sign On Settings.
+There are two ways to set up your custom provider: using the Buildkite meta-data XML to your identity provider, or manually adding your Buildkite data into your identity provider. 
 
-Manual SSO provisioning in Buildkite is available via our GraphQL explorer. For further information, see the [GraphQL SSO Setup Guide](/docs/integrations/sso/sso-setup-with-graphql). 
+After setting up your provider, you'll need to set up your SSO provider in your Buildkite organization's Single Sign On Settings.
 
 <div class="Docs__note">
-<p class="Docs__note__heading">Before starting your provider setup</p>
-<p>Email the Buildkite team at support@buildkite.com to request your unique SSO service URL, you will need this during set up.</p>
+  <p>You can also set up SSO providers manually with GraphQL. See the <a href="/docs/integrations/sso/sso-setup-with-graphql">SSO Setup with GraphQL Guide</a> for detailed instructions and code samples.</p>
 </div>
 
-### Step 1. Option A - Setting up with meta-data
+## Option A - Setting up your IdP using a meta-data URL or ACS URL
```

Comment:
Yeah, like you mentioned, this nav/structure is getting pretty wild!

<img width="299" alt="image" src="https://user-images.githubusercontent.com/153/72115309-b998cb00-339a-11ea-863d-51e47d81a05c.png">

What if we made the "Option A/B"'s third-level headings?

```suggestion
### Option A - Setting up your IdP using a meta-data URL or ACS URL
```

Reply:
I wanted the different options to be visible up front/as soon as you get to the page, so I was hoping to keep it in the TOC somehow... but I don't see how that's possible. Do you think we could figure out some way of showing people that there's two ways to do things that's not the TOC? Or just leave them to scroll and figure it out? Could it be solved by another line of text in that top intro paragraph section before the TOC?

Reply:
@harrietgrace want to have a quick chat about it? Maybe if we talk it through we'll have some other ideas… at the moment I'm stumped too.